src/Test/Plutip/Config.hs

```diff
@@ -1,11 +1,14 @@
 module Test.Plutip.Config (
   PlutipConfig (..),
+  WorkingDirectory (..),
 ) where
 
 import Data.Default (Default, def)
 import GHC.Generics (Generic)
 import GHC.Natural (Natural)
 
+data WorkingDirectory = Temporary | Fixed {path :: FilePath, shouldKeep :: Bool}
```

Comment:
The definition itself is pretty self-explanatory, but maybe it is till worth to add documentation.

Reply:
Have done :)